src/modules/staffAssignment/components/YourClients.tsx

```diff
@@ -101,19 +101,21 @@ const MY_CLIENTS_COLUMNS: ColumnDef<StaffAssignmentWithClientsFragment>[] = [
   },
 ];
 
-const MyClients = () => {
+const YourClients = () => {
   const { user: currentUser } = useAuth();
   if (!currentUser) return;
 
   return (
     <>
-      <Typography component='h2' variant='h4'>
-        My Clients
-      </Typography>
-      <Typography variant='body1'>
-        Households with active enrollments assigned to you
-      </Typography>
       <Paper>
+        <Stack gap={1} m={2}>
+          <Typography component='h2' variant='h4'>
+            Your Clients
+          </Typography>
+          <Typography variant='caption'>
+            Households with active enrollments assigned to you
+          </Typography>
+        </Stack>
```

Comment:
Suggestion:
```jsx
      <CommonCard title='Your Clients' padContent={false}>
        <Typography variant='body2' sx={{ px: 2 }}>
          Households with active enrollments assigned to you
        </Typography>
```
- discussed at standup that subtitle was supposed to be body2 (figma did list caption)
- per other comment, use a shared card component